fe/fe-core/src/main/java/com/starrocks/credential/empty/EmptyCloudConfiguration.java

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.credential.empty;
+
+import com.starrocks.credential.CloudConfiguration;
+import com.starrocks.thrift.TCloudConfiguration;
+import org.apache.hadoop.conf.Configuration;
+
+// If user didn't specific any credential, we create EmptyCloudConfiguration instead.
+// It will use Hadoop default constructor instead, user can put core-site.xml into java CLASSPATH to control
+// authentication manually
+public class EmptyCloudConfiguration implements CloudConfiguration {
```

Comment:
```suggestion
public static CloudConfiguration EMPTY() {
        return new CloudConfiguration() {
            @Override
            public void toThrift(TCloudConfiguration tCloudConfiguration) {

            }

            @Override
            public void applyToConfiguration(Configuration configuration) {

            }

            @Override
            public String getCredentialString() {
                return "empty";
            }
        };
    }
```
may be you could port it in the CloudConfigurationFactory

Reply:
it could be integrated into tryBuildForStorage()

Reply:
Improved.